views/incubator/omniledger.tpl

```diff
@@ -0,0 +1,101 @@
+    can be a good solution.
+    Examples of this include money transfers if you don't trust any bank, or authentication to services
+    without trusting only one (like Google or Facebook).<br>
+    Three important elements of blockchains are:
+</p>
+
+<table border="0">
+    <tr style="vertical-align: top;">
+        <td width="33%" style="padding: 1em;">
+            <h3>Decentralization</h3>
+            <img src="/resources/incubator/images/omniledger/decentralized.png" width="33%"
+                 style="float: left; padding: 1em;"/>
+            Blockchains got a lot of publicity under the name of <strong>Bitcoin</strong>.
+            For some it is a source of great wealth, for others it is a synonym for a lot
+            of wasted energy. Few remember the origin of Bitcoin, which is about
+            <strong>decentralization</strong>.
+            Decentralization means that there is no central entity, but that the responsibility
+            is spread out over several entities. In the example of Switzerland, one could
+            say that the organization in cantons leads to a decentralized government.
+        </td>
+        <td width="33%" style="padding: 1em;">
+            <h3>Transactions</h3>
+            <img src="/resources/incubator/images/omniledger/transaction_general.png" width="33%"
+                 style="float: left; padding: 1em;"/>
+            <img src="/resources/incubator/images/omniledger/transaction_money.png" width="33%"
+                 style="float: right; padding: 1em;"/>
+            A transaction is what is stored in a blockchain.
+            In the case of Bitcoin, transactions describe the <strong>movement of money</strong>
+            between different parties.
+            Instead of having a central bank deciding which money transfers are valid,
+            Bitcoin spreads this decision over many computers which agree together which transfers are valid.
+            A new kind of transaction that is gaining traction in 2021 are Non Fungible Tokens, or NFTs,
+            which represent the digital ownership of a physical object.
+        </td>
+        <td width="33%" style="padding: 1em;">
+            <h3>Consensus</h3>
+            <img src="/resources/incubator/images/omniledger/distributed-ledger.png" width="33%"
+                 style="float: left; padding: 1em;"/>
+            There are other blockchains than Bitcoin, but they all solve the problem of
+            <strong>consensus</strong> for decentralized systems. Reaching a consensus between
+            different entities is not easy, as some of the entities can try to cheat.
+            So the conception of blockchains always takes into account that some of the
+            participants will try to cheat. If too many are cheating, the consensus will fail,
+            or invalid transactions might get accepted, leading to a failure of the
+            decentralization.
```

Comment:
hum, as we are introducing AT2 which provides (nearly) the same capabilities as Bitcoin, and which doesn't need consensus, maybe find another property? otherwise, it might be hard to have a coherent corpus

Reply:
good question - what is a blockchain? For one, there should be blocks that are chained together, which is not how AT2 works, if I understand correctly. But Avalanche doesn't do blocks, neither, bug DAGs, and they still call it blockchain.

To me it looks OK to say AT2 is a special blockchain as it does only transactions, and for those a full consensus is not needed. And most blockchains _do_ have a consensus mechanism. Even those who don't need it (bitcoin ;).

So I prefer keeping it in here. What do you think?

Reply:
> good question - what is a blockchain? For one, there should be blocks that are chained together

From Wikipedia: A blockchain is a growing list of records, called blocks, that are linked using cryptography. So `git` is a blockchain :stuck_out_tongue_closed_eyes:
"distributed ledger" looks better, it describe the features of the technology and not an implementation detail. Of course, as many people know the former, it would be good to say somewhere that's is the "old" word used to describe it. Then, you can even say that "consensuss" is a specificity of OL, to provide the security behind the distributed part of the ledger.

> which is not how AT2 works, if I understand correctly.

You're right, no need to have an global ordering of the transaction (so concensus), only for a single sender (local consensus of size one).

> But Avalanche doesn't do blocks, neither, bug DAGs, and they still call it blockchain.

Marketing, they jumped on the hype train. And you can kinda see it as a/many blockchain if you only see the leafs.

> To me it looks OK to say AT2 is a special blockchain as it does only transactions, and for those a full consensus is not needed. And most blockchains _do_ have a consensus mechanism. Even those who don't need it (bitcoin ;).

Having a special-smth is not really descriptive IMO, that's why I'm more for the "distributed ledger" explaination, as it is correct basis: OL implements it with consensus and can do asset transferts & smart contracts, AT2 implements with magic-math and can only transfer assets.

> So I prefer keeping it in here. What do you think?

Your call, there isn't really a good way to decide between the two, as the hierarchy of theses pages are flat, there is not really way to have first an introduction on distributed ledger and two implementations, so I think that anyone will read each presentation in full, and not really think much of the differences between the two definitions.

Reply:
Well, Wikipedia adds `consensus` as needed for a `Distributed Ledger`, so that would not fit AT2. And the `Ledger` should refer mostly to `UTXO`s and should theoretically exclude account-based solutions, like Ethereum and OmniLedger.

So it's a mess. Also, when people talk about `DLT`s, they include Hyperledger Fabric, which is really more of a distributed database with some smart contracts attached to it.

I changed the phrase before the triplet. Let's go with that.